docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,272 @@
+## Overview
+
+Registering a WEC with an ITS follows the same process as registering a managed cluster with an OCM hub. KubeStellar uses Open Cluster Management (OCM) for cluster registration and management.
+
+For the complete OCM registration process, refer to the [official Open Cluster Management documentation](https://open-cluster-management.io/docs/getting-started/installation/register-a-cluster/). The instructions below provide a complete registration process that is more focused on KubeStellar-specific considerations.
+
+### Terminology Mapping
+
+| OCM Term             | KubeStellar Equivalent                        |
+|----------------------|-----------------------------------------------|
+| **OCM Hub**          | **ITS** (Inventory and Transport Space) |
+| **OCM Managed Cluster** | **WEC** (Workload Execution Cluster) |
+| **Klusterlet**       | **Klusterlet** |
+
+## Prerequisites
+
+Before registering a WEC, ensure you have:
+
+1. **A running Kubernetes cluster** that will serve as the WEC
+2. **Network connectivity** from the WEC to the KubeFlex hosting cluster (HTTPS connections must be possible)
+3. **`kubectl` access** to both the WEC and ITS
+4. **`clusteradm` CLI tool** installed on the machine where you'll run the registration commands ([installation guide](https://open-cluster-management.io/docs/getting-started/installation/start-the-control-plane/))
+5. **An existing ITS** with OCM cluster manager running
+6. **Sufficient permissions** to create resources in both clusters (typically cluster-admin or equivalent)
+
+To verify your ITS is ready for WEC registration:
+
+```shell
+# Check if the ITS is accessible (this returns a short result)
+kubectl --context <its-context> get ServiceAccount default
+
+# Verify OCM cluster manager is running (look for absence of Pending or failing Pods)
+kubectl --context <its-context> get pods -n open-cluster-management-hub
+
+# Check for the customization-properties namespace
+kubectl --context <its-context> get ns customization-properties
+```
+
+## Registration Process
+
+### Step 1: Get the Registration Token
+
+Obtain a registration token from the ITS:
+
+```shell
+# Get the registration token from the ITS
+clusteradm --context <its-context> get token
+```
+
+This command outputs a `clusteradm join` command that you'll use in the next step.
+
+### Step 2: Join the WEC to the ITS
+
+Execute the join command on the WEC to initiate the registration process:
+
+```shell
+# Basic join command (additional flags may be required based on your environment)
+clusteradm join --hub-token <token> --hub-apiserver <api-server-url> --cluster-name <your-wec-name> --context <wec-context>
+```
+
+**Important flags:**
+- `--cluster-name`: Choose a unique name for your WEC
+- `--context`: Specify the kubectl context for your WEC
+- `--singleton`: Use this flag if the WEC is a single-node cluster
+- `--force-internal-endpoint-lookup`: Required for Kind clusters and other local development setups
```

Comment:
This does not render as you intended. Make a practice of proof-reading your rendering.

<img width="817" height="87" alt="image" src="https://github.com/user-attachments/assets/a8b99e72-38cd-4511-b084-a75616e33c01" />

Reply:
This is still not rendering as you intend. Compare the website preview

<img width="760" height="118" alt="image" src="https://github.com/user-attachments/assets/110b1745-a326-4e41-ac13-4566a984e977" />

with the direct view at GitHub:

<img width="725" height="180" alt="image" src="https://github.com/user-attachments/assets/55236bc6-6441-4b86-bb8d-b16d23ae29ff" />